File: jcu-zoo/src/adminUI/MySQLConnection.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package adminUI;

import com.mysql.jdbc.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.util.logging.Level;
import java.util.logging.Logger;

/**
 *
 * @author Panda
 */
public class MySQLConnection { 
    
     public static Connection connectDB() {
        Connection connection = null;
        try {
            String url = "jdbc:mysql://" + Configuration.hostname + "/" + Configuration.databaseName;
            Class.forName("com.mysql.jdbc.Driver").newInstance();
            connection = (Connection) DriverManager.getConnection(url, Configuration.dbUser, Configuration.dbPassword);

            System.out.println("Database connection established");
            return connection;
        } catch (Exception ex) {
            Logger.getLogger(MySQLConnection.class.getName()).log(Level.SEVERE, null, ex);
        }
        return null;
    }
     public static void main(String[] args) {
        Configuration.loadConfigurationFromFile();
        Connection conn = MySQLConnection.connectDB();
        try {
            conn.close();
        } catch (SQLException ex) {
            Logger.getLogger(MySQLConnection.class.getName()).log(Level.SEVERE, null, ex);
        }
    }
     //private static void disconnectDB();
}

File: jcu-zoo-php/class/MySQLConnection.php
<?php

/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of MySQLConnection
 *
 * @author Panda
 */
class MySQLConnection {
   private $dbh;
   protected $user = "test";
   protected $pass = "test";
   protected $dbName = "jcu-zoo";
   public function __construct(){
       try{
           $this->dbh = new PDO('mysql:host=localhost;dbname='.$this->dbName, $this->user, $this->pass); 
       } catch (Exception $e){
            echo("Error connecting to database");
            die($e);
}
   }
   public function getConnection(){
       return $this->dbh;
   }

}

?>
